src/WebJobs.Script/Config/ScriptTelemetryProcessor.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.ApplicationInsights.Channel;
+using Microsoft.ApplicationInsights.DataContracts;
+using Microsoft.ApplicationInsights.Extensibility;
+using Microsoft.Azure.WebJobs.Script.Workers.Rpc;
+
+namespace Microsoft.Azure.WebJobs.Script.Config
+{
+    internal class ScriptTelemetryProcessor : ITelemetryProcessor
+    {
+        public ScriptTelemetryProcessor(ITelemetryProcessor next)
+        {
+            this.Next = next;
+        }
+
+        private ITelemetryProcessor Next { get; set; }
+
+        public void Process(ITelemetry item)
+        {
+            if (FeatureFlags.IsEnabled(ScriptConstants.FeatureFlagEnableUserCodeException)
+                && item is ExceptionTelemetry exceptionTelemetry
+                && exceptionTelemetry.Exception.InnerException is RpcException rpcException
```

Comment:
nit: do we need some extra null checks here? 

```suggestion
                && exceptionTelemetry?.Exception?.InnerException is RpcException rpcException
```